### PR TITLE
fix: move `holder` to top level in the sidebar

### PR DESCRIPTION
**Pull request by therealharpaljadeja**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -120,17 +120,18 @@ module.exports = {
...
-              // position: "left",
             },
             {
               to: "protocol/",
               label: "Protocol",
-              // position: "left",
+            },
+            {
+              to: "holder/",
+              label: "Holder",
```
Comment (lvpeschke) [inline docusaurus.config.js:130]: I think I would have expected the `Holder` section to be at the same level as `Developer` and `Validator`, wdyt?
Especially judging by the content on https://docs.celo.org/holder, which reads like a big entry page.

Comment (therealharpaljadeja) [reply docusaurus.config.js:130]: `Developer` and `Holder` are at the same level we just tucked `Holder` into the more part in order to have a better mobile responsible navbar.

If we want `Holder` to be outside the dropdown then we have to trade it with `Validator` or `What is Celo` item.

image

image

Comment (lvpeschke) [reply docusaurus.config.js:130]: That makes sense. I wouldn't trade it for one of the other options then.
How about making it the first in the dropdown?

Comment (therealharpaljadeja) [reply docusaurus.config.js]: Done!

**Review by lvpeschke — APPROVED** [review]

**Pull request merged**